src/components/TableCard.tsx

```diff
@@ -0,0 +1,100 @@
+import { ReactNode, useMemo } from "react";
+import { useHover } from "react-aria";
+import { Css, Palette } from "src/Css";
+import { ButtonMenu, MenuItem } from "./ButtonMenu";
+import { Tag, TagType } from "./Tag";
+
+export type TableCardType = "card" | "list";
+
+type CardTag = {
+  text: string;
+  type?: TagType;
+};
+
+export interface TableCardProps {
+  title: string;
+  subtitle: string;
+  detailContent?: ReactNode;
+  imgSrc: string;
```

Comment:
@blambillotte @khalidwilliams 

I just noticed that I didn't handle the empty state. I just wanted to double check, but I suppose that I should update this card to handle that state as well?

<img width="589" alt="image" src="https://github.com/user-attachments/assets/bede78ad-0366-4a6b-b377-a25e7793ff23" />

Reply:
Hmm I missed that too. If it's not too much time / work to add in it's probably worth it, but I don't think any of our team's work requires the empty state, so it would be fine to add in a follow up later IMO.

Reply:
Ehh maybe leave for whoever needs it next? IMO the null state is more of a separate component than actually being a null state of the card itself since it doesn't really share any properties other than being sized similar?

Reply:
Yeah I agree, I wasn't sure if it made sense to include or not. Thanks for the input!